tests/FileSystemTests.cpp

```diff
@@ -7,6 +7,8 @@
 #include <string>
 #endif
 
+#include <thread>

```

Comment:
I'm a little surprised this builds (though maybe it gives the same warnings as other code). This should ideally be moved up inside of the `#ifdef WIL_ENABLE_EXCEPTIONS` above

Reply:
fixed